web/src/lib/i18n/en.json

```
@@ -557,6 +608,7 @@
   "immich_web_interface": "Immich Web Interface",
   "import_from_json": "Import from JSON",
   "import_path": "Import path",
+  "in_albums": "In {count} {count, plural, one {album} other {albums}}",
```

Just to get proper number formatting

```suggestion
  "in_albums": "In {count, plural, one {# album} other {# albums}}",
```